src/core/evaluator.js

```diff
@@ -1676,6 +1677,9 @@ class PartialEvaluator {
               const localImage = localImageCache.getByName(name);
               if (localImage) {
                 operatorList.addOp(localImage.fn, localImage.args);
+                if (localImage.args.length !== 0 && localImage.args[0].count) {
```

Comment:
Here, and similarly below, this will check for a `count` property even for OPS where that doesn't make sense. For "normal" images, note how the *first* argument is a string.

Hence, please change this to be explicit about what's intended in these cases:
```javascript
if (
  localImage.fn === OPS.paintImageMaskXObject &&
  localImage.args[0] &&
  localImage.args[0].count > 0
) {
```